avocado/core/exit_codes.py

```diff
@@ -21,21 +21,22 @@
 """
 
 #: Both job and tests PASSed
-AVOCADO_ALL_OK = 0
+AVOCADO_ALL_OK = 0x000
```

Comment:
There was a card doing that. Btw it'd require further changes to documentation and the framework itself to really report the right exit status.